processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -478,14 +547,10 @@ func (c *WatchClient) extractPodAttributes(pod *api_v1.Pod) map[string]string {
 	if c.Rules.StartTime {
 		ts := pod.GetCreationTimestamp()
 		if !ts.IsZero() {
-			if enableRFC3339Timestamp.IsEnabled() {
```

Comment:
the feature gate is already in stable phase, therefore there is no possibility to disable it, therefore this `if-else` statement is unneeded, as the `else` branch is unreachable

Reply:
can we just remove the feature gate?

Reply:
opening https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38810 for your review.